pkgs/development/python-modules/git-find-repos/default.nix

```
@@ -0,0 +1,27 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools-scm,
+}:
+buildPythonPackage rec {
+  pname = "git-find-repos";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "acroz";
+    repo = "git-find-repos";
+    rev = version;
+    sha256 = "sha256-4TuZlt6XH4//DBHPuIMl/i3Tp6Uft62dGCTAuZ2rseE=";
+  };
+
+  build-system = [setuptools-scm];
+
+  meta = with lib; {
+    description = "A simple CLI tool for finding git repositories.";
+    homepage = "https://github.com/acroz/git-find-repos";
+    license = licenses.mit;
+    maintainers = with maintainers; [yajo];
+  };
```

```suggestion
  meta = {
    description = "A simple CLI tool for finding git repositories.";
    homepage = "https://github.com/acroz/git-find-repos";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ yajo ];
  };
```

The use of `with lib;` is discouraged. ref: https://github.com/NixOS/nixpkgs/pull/293767
You can put `lib.licenses.mit` in `meta.license`, and others too.